ci/jjb/jobs/pulp3-smash-runner.yaml

```diff
@@ -97,6 +103,23 @@
             git clone https://github.com/pulp/pulp_rpm.git --branch master
             git clone https://github.com/pulp/pulp_docker.git --branch master
 
+            # Configure podman for docker tests 
+            sudo yum install -y podman
```

Comment:
It is a fedora box?
```suggestion
            sudo dnf install -y podman
```

Reply:
@kersommoura I would like to keep `yum` as it is agnostic. If someday we use RHEL it will still work. And on fedora `yum` is a kind of `alias` to `dnf`

This solution is already applied to Jenkins and workds see build 85